charts/rancher-vsphere-cpi/values.yaml

```
@@ -27,11 +27,16 @@ vCenter:
 # - On running a helm template, Helm generally assumes the kubeVersion is v1.20.0
 # - On running a helm install --dry-run, the correct kubeVersion should be chosen.
 versionOverrides:
-  - constraint: ">= 1.24 < 1.26"
+  - constraint: "~ 1.25"
+      values:
+        cloudControllerManager:
+          repository: rancher/mirrored-cloud-provider-vsphere-cpi-release-manager
+          tag: v1.25.0
```

These are over-indented, they should be at the same level as the constraint since the values are part of the same map
```suggestion
    values:
      cloudControllerManager:
        repository: rancher/mirrored-cloud-provider-vsphere-cpi-release-manager
        tag: v1.25.0
```